code_generation/quantities/output.py

```diff
@@ -122,8 +123,12 @@
 gen_m_vis = Quantity("gen_m_vis")
 isoWeight_1 = Quantity("isoWeight_1")
 isoWeight_2 = Quantity("isoWeight_2")
+isoWeight_old_1 = Quantity("isoWeight_old_1")
```

Comment:
i would remove those, just keep the quantity in once, but we keep both functions to do the same. I should be clear to a user, that extracting some weights twice is not required

Reply:
Done